paywall/stories/Paywall.stories.js

```diff
@@ -79,6 +80,32 @@ const config = {
   },
 }
 
+function FakeItTillYouMakeIt({ children }) {
+  const divit = useRef()
+  return (
+    <div
+      ref={divit}
+      style={{
+        position: 'absolute',
```

Comment:
Do we have a plan for keeping the storybook in sync with the paywall flag? (thinking of what happens if we change some of the surrounding stuff that isn't part of the flag itself, which we mock here)

LG otherwise!

Reply:
I am unsure how we could best do this. We could make a parameter to the Paywall which allows us to use that for setting styles instead of the `window.document.body` but we will always have to fake the stuff that happens in `paywall.min.js`

Reply:
(ideas appreciated!)